dipy/io/utils.py

```diff
@@ -91,3 +91,42 @@ def decfa(img_orig, scale=False):
     new_hdr.set_data_dtype(dest_dtype)
 
     return Nifti1Image(out_data, affine=img_orig.affine, header=new_hdr)
+
+
+def decfa_to_float(img_orig):
+    """
+    Create a nifti-compliant directional-encoded color FA file.
+
+    Parameters
+    ----------
+    img_orig : Nifti1Image class instance.
+        Contains encoding of the DEC FA image with a 3D volume of data, where
+        each element is a (R, G, B) tuple in uint8.
+
+    Returns
+    -------
+    img : Nifti1Image class instance with uint8 dtype.
+
+
+    Notes
+    -----
+    For a description of this format, see:
+
+    https://nifti.nimh.nih.gov/nifti-1/documentation/nifti1fields/nifti1fields_pages/datatype.html
```

Comment:
Is it really there? This format is the classical format used by most software, but is not in the standard.

Reply:
I am not sure what you mean by "the standard". 

The triplet format is on this wiki page as "DT_RGB".

Reply:
True. Went too fast.